File: src/main/java/no/nav/syfo/consumer/rest/Ident.java
package no.nav.syfo.consumer.rest;

import lombok.Builder;
import lombok.Getter;
import lombok.Value;

import java.util.List;

@Value
@Builder
@Getter
class Ident {
    private String ident;
    private String identgruppe;
    private Boolean gjeldende;
}

@Value
@Builder
@Getter
class AktorResponse {
    private List<Ident> identer;
    private String feilmelding;
}

File: src/main/java/no/nav/syfo/consumer/rest/aktor/Ident.kt
package no.nav.syfo.consumer.rest.aktor;

import lombok.Builder;
import lombok.Getter;
import lombok.Value;

import java.util.HashMap;
import java.util.List;

@Value
@Builder
@Getter
public class Ident {
    private String ident;
    private String identgruppe;
    private Boolean gjeldende;
}
